src/predictor/gpu_predictor.cu

```diff
@@ -1048,11 +1044,8 @@ class GPUPredictor : public xgboost::Predictor {
     if (tree_weights != nullptr) {
       LOG(FATAL) << "Dart booster feature " << not_implemented;
     }
-    CHECK(!p_fmat->Info().IsColumnSplit())
-        << "Predict contribution support for column-wise data split is not yet implemented.";
```

Comment:
Is SHAP support available for column-wise split data now?

Reply:
The check was removed between PR rebases. I will bring it back.